src/tomorrow_io/CMakeLists.txt

```
@@ -0,0 +1,21 @@
+# (C) Copyright 2024 The Tomorrow Companies, Inc.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+add_executable(convert_tio.x
+	creation.cc
+	creation_tio_sat_inst_specs.cc
+	copy.h
+	copy_function.cc
+	copy_helpers.cc
+	copy_tms.cc
+	copy_tms_datetime.cc
+	detect_tms_type.cc
+	product.cc
+	product.h
+	main_tio.cc
+	)
+target_link_libraries(convert_tio.x PUBLIC ioda)
```

Your CI test setup is a bit different from mine, which is why the two tests are failing with:
```
/workdir/test_root/build/bin/convert_tio.x: No such file or directory
```

The CMake line should look something like this:

```suggestion
ecbuild_add_executable( TARGET convert_tio.x
	SOURCES
	  creation.cc
	  creation_tio_sat_inst_specs.cc
	  copy.h
	  copy_function.cc
	  copy_helpers.cc
	  copy_tms.cc
	  copy_tms_datetime.cc
	  detect_tms_type.cc
	  product.cc
	  product.h
	  main_tio.cc
	LIBS
	  ioda
	)
```